src/commonmark/en/content/user/configure-programs-in-the-maintenance-app.md

```diff
@@ -1248,7 +1248,7 @@ A filter that uses the "Reporting period end" variable to only include
 women who has an LMP that would be in the first
```

Comment:
Need to change the quoted variable in this sentence.

Reply:
Fixed